tests/vmi_multus_test.go

```diff
@@ -856,10 +864,15 @@ var _ = Describe("[Serial]SRIOV", func() {
 			waitVmi(vmi1)
 			waitVmi(vmi2)
 
+			vmi1, err = virtClient.VirtualMachineInstance(vmi1.Namespace).Get(vmi1.Name, &metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			vmi2, err = virtClient.VirtualMachineInstance(vmi2.Namespace).Get(vmi2.Name, &metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+
 			// manually configure IP/link on sriov interfaces because there is
 			// no DHCP server to serve the address to the guest
-			configInterface(vmi1, "eth1", cidrA)
-			configInterface(vmi2, "eth1", cidrB)
+			Expect(configSRIOVInterface(virtClient, vmi1, mac1.String(), cidrA)).To(Succeed())
```

Comment:
the error returning from `configSRIOVInterface` already contains all that info except for the IP. I'll added it there. will that be sufficient?

Reply:
If the VMI is nil, we better propagate such error so we could see it in the logs later.
I am not 100% sure that if you use the combination of To(Succeed()) will print the error from `configSRIOVInterface`
It would?

Reply:
> If the VMI is nil, we better propagate such error so we could see it in the logs later.

I'll add this info to the error it self then, and it will show up in the logs.

> I am not 100% sure that if you use the combination of To(Succeed()) will print the error from configSRIOVInterface
It would?

I verified it and it does:) 

```
• Failure [56.940 seconds]
[Serial]SRIOV
tests/vmi_multus_test.go:571
  VirtualMachineInstance with sriov plugin interface
  tests/vmi_multus_test.go:627
    [test_id:3956]should connect to another machine with sriov interface over IPv4 [It]
    tests/vmi_multus_test.go:882

    Expected success, but got an error:
        <*fmt.wrapError | 0xc003a92360>: {
            msg: "Failed to configure address this_is_not_leagal_address_:( for interface eth0 on VMI testvmix4g9krrq729kdfj27cjhnzswhn275dttzmv4mz2tvcxflmms: expect: timer expired after 15 seconds",
            err: 15000000000,
        }
        Failed to configure address this_is_not_leagal_address_:( for interface eth0 on VMI testvmix4g9krrq729kdfj27cjhnzswhn275dttzmv4mz2tvcxflmms: expect: timer expired after 15 seconds

```